src/test/java/net/snowflake/ingest/streaming/internal/it/ColumnNamesIT.java

```diff
@@ -252,7 +297,12 @@ private void testInsertRowFails(
   private String createSimpleTable(String createTableColumnName) throws SQLException {
     String tableName = "a" + UUID.randomUUID().toString().replace("-", "_");
     String createTableSql =
-        String.format("create table %s (%s int);", tableName, createTableColumnName);
+        String.format(
+            "create %s table %s (%s int) %s",
+            enableIcebergStreaming ? "iceberg" : "",
+            tableName,
+            createTableColumnName,
+            enableIcebergStreaming ? getIcebergTableConfig(tableName) : "");
```

Comment:
IMO we should refactor this to have a method that takes in the table name and column schema list only, and adds the proper prefix (iceberg table vs. table) and proper suffix (catalog and external_volume and base_location).

Instead of every create table call having to worry about both these things.

If this PR is good to merge am ok with this test refactoring in the next PR. Please also pick up any deferred comments from the last PR if any!